webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala

```diff
@@ -782,7 +786,7 @@ class OntologyResponderV2(responderData: ResponderData) extends Responder(respon
 
         currentTime: Instant = Instant.now
 
-        updateSparql = org.knora.webapi.messages.twirl.queries.sparql.v2.txt
+        updateSparql = v2.txt
```

Comment:
I understand and now have changed the import to be `org.knora.webapi.messages._` then, because for me the fully qualified package is too much of visual clutter and noise making it harder to read.

0b36bf4ba939471a437abcbbfb726c17f683282a